### PR TITLE
pdfa: provided first PDF/A examples

### PR DESCRIPTION
**Pull request by kucjac**

### CODE REVIEW

File: pdfa/pdf_apply_standard.go [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+	start := time.Now()
+
+	// Create reader.
+	inputFile, err := os.Open(inputPath)
+	if err != nil {
+		log.Fatalf("Fail: %v\n", err)
+	}
+	defer inputFile.Close()
+
+	reader, err := model.NewPdfReader(inputFile)
```
Comment (gunnsth) [inline pdfa/pdf_apply_standard.go:54]: can use model.NewPdfReaderFromFile(inputFile)... slightly shorter :).
So we don't need to open the reader in compliance mode to apply?  ok cool

File: pdfa/pdf_apply_standard.go [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+	}
+
+	// Generate a PDFWriter from PDFReader.
+	pdfWriter, err := reader.ToWriter(nil)
+	if err != nil {
+		log.Fatalf("Fail: %v\n", err)
+	}
+
+	// Apply standard PDF/A-1B.
+	pdfWriter.ApplyStandard(pdfa.NewProfile1B(nil))
```
Comment (gunnsth) [inline pdfa/pdf_apply_standard.go:66]: This reads very well.
Great

File: pdfa/pdf_validate_standard.go [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+	start := time.Now()
+
+	// Create reader.
+	inputFile, err := os.Open(inputPath)
+	if err != nil {
+		log.Fatalf("Fail: %v\n", err)
+	}
+	defer inputFile.Close()
+
+	detailedReader, err := model.NewCompliancePdfReader(inputFile)
```
Comment (gunnsth) [inline pdfa/pdf_validate_standard.go:53]: How about the `compliance` flag on the reader options... Are we not using thaT?

**Review by gunnsth — COMMENTED** [review]
Few comments/questions to go with review of https://github.com/unidoc/unipdf-src/pull/185#pullrequestreview-728451958

File: pdfa/pdf_apply_standard.go (hunk 1 above)
Comment (kucjac) [reply pdfa/pdf_apply_standard.go]: Sure :+1: 
Done. Yeah, it is not required by the writer. It mostly is used by the PdfReader to extract some metadata from the document.

File: pdfa/pdf_validate_standard.go (hunk 3 above)
Comment (kucjac) [reply pdfa/pdf_validate_standard.go:53]: The `NewCompliancePdfReader` is creating new `PdfReader` with the `ComplianceMode` flag set to true.

**Pull request merged**